src/options.js

```diff
@@ -1,3 +1,13 @@
+import { isUndefined } from './utils.js';
+
+
+function getCookieDomain() {
+  return isUndefined(window) ? '' : `${window.location.hostname}`;
+}
+
+function getCookieDomainUrl(path = '') {
```

Comment:
Contextually, you are not returning `CookieDomainUrl` but `redirectUri`, so `getRedirectUri` would be a much better name for this function.